docs/cli-next/action.yml

```diff
@@ -20,25 +20,21 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - if: ${{ github.event_name == 'pull_request' }}
-      uses: ory/ci/checkout@master
-      with:
-        path: current-repo
-        fetch-depth: 0
-        ref: ${{ github.event.pull_request.head.ref }}
-        repository: ${{ github.event.pull_request.head.repo.full_name }}
-        token: ${{ inputs.token || github.token }}
-    - if: ${{ github.event_name != 'pull_request' }}
-      uses: ory/ci/checkout@master
+    - if: ${{ github.event_name != 'push' }}
+      shell: bash
+      run: |
+        echo "This workflow is only meant to be triggered by pushes to specific branches (i.e. master)."
+        exit 1
+    - uses: ory/ci/checkout@master
       with:
         token: ${{ inputs.token || github.token }}
         path: current-repo
         fetch-depth: 0
     - uses: actions/setup-go@v2
       with:
-        go-version: ^1.16
+        go-version: ^1.18
     - run: |
-        git config --global user.email "60093411+ory-bot@users.noreply.github.com"
+        git config --global user.email "ory-bot@users.noreply.github.com"
```

Comment:
What is the point in adding that random number?

Reply:
It's the github user id. Since the username can change. It's used to identify what account the email belongs to when doing the git commit. It's coming from the github user interface.